SwiftyBluetooth/Source/Central.swift

```diff
@@ -62,6 +62,18 @@ public enum PeripheralScanResult {
 }
 
 /**
+ The different results returned in the closure of the Central connect(...) and disconnect(...) functions.
```

Comment:
No, I want to use a Result<T> type struct in the future for the callback results instead of the current optional Result/Error duo, but I haven't gotten to it yet :) But in the meantime, I don't want an out of place enum type for 1 single particular function when it will be replaced by a Result<T> in the future for consistency sake.